docs/azure.rst

```diff
@@ -238,19 +238,20 @@ Private container registry
 As of version ``21.05.0-edge``, a private container registry from where to pull Docker images can be optionally specified as follows ::
 
     azure {
-        batch {
-            registry {
-                server =  '<YOUR REGISTRY SERVER>' // e.g.: docker.io, quay.io, <ACCOUNT>.azurecr.io, etc.
-                userName =  '<YOUR REGISTRY USER NAME>'
-                password =  '<YOUR REGISTRY PASSWORD>'
-            }
-        }
+		registry {
+			server =  '<YOUR REGISTRY SERVER>' // e.g.: docker.io, quay.io, <ACCOUNT>.azurecr.io, etc.
+			userName =  '<YOUR REGISTRY USER NAME>'
+			password =  '<YOUR REGISTRY PASSWORD>'
+		}
     }
```

Comment:
\<pedantic mode>

  let's indent `registry` as it was `batch`

\</pedantic mode> 😄

Reply:
<keen mode> It was good on my Idea editor </keen mode>
<complaining mode> thanks github </mode>
<collaborative mode>fixed</collaborative mode>

Reply:
No tabs! 😉